chore(azure-functions-linux): update README.md (#4281)
prisma/ecosystem-tests: platforms-serverless/azure-functions-linux/README.md

### Prepare

To create a function on your own account, run `sh create.sh` first.

### Cleaning up 

If you see an error like `ERROR: Number of sites in server farm 'WestEuropeLinuxDynamicPlan' exceeds the maximum allowed for 'Dynamic' SKU.`, you will need to delete old functions that failed and were not automatically cleaned up.

> This is a very entertaining thing to do, you will meet a 🦥 API and party of rate limits 😉

Author: ```suggestion
This is a very entertaining thing to do, you will meet a 🦥 API and a party of rate limits 😉
```

> 

Author: ````suggestion

There is one command that requires to have the `fish` shell installed. You can find instructions at https://fishshell.com/.
For macOS: `brew install fish`
For Ubuntu:
```
sudo apt-add-repository ppa:fish-shell/release-3
sudo apt update
sudo apt install fish
```
````

```sh
az login

> # List all functions ids
> az functionapp list --resource-group prisma-e2e-linux --query "[].id" --output tsv

Author: ```suggestion
# List all functions ids and count them
E2E_AZ_LINUX=$(az functionapp list --resource-group prisma-e2e-linux --query "[].id" --output tsv) && echo $E2E_AZ_LINUX && echo $E2E_AZ_LINUX | wc -l
```

> # Note that the following command will work in `fish` terminal specifically

Author: ```suggestion
# Note that the next command will work in `fish` shell specifically
# So we need to start the fish shell now
```

### Prepare

To create a function on your own account, run `sh create.sh` first.

### Cleaning up 

> If you see an error like `ERROR: Number of sites in server farm 'WestEuropeLinuxDynamicPlan' exceeds the maximum allowed for 'Dynamic' SKU.`
> You will need to delete old functions that failed and were not automatically cleaned up.

Reviewer: ```suggestion
If you see an error like `ERROR: Number of sites in server farm 'WestEuropeLinuxDynamicPlan' exceeds the maximum allowed for 'Dynamic' SKU.`, you will need to delete old functions that failed and were not automatically cleaned up.
```
